tensorbay/client/version.py

```diff
@@ -831,7 +831,6 @@ def create_job(
             "conjunction": conjunction,
             "filters": filters,
             "unit": unit,
```

Comment:
commit message:
`delete the "title" argument in BasicSearch.create_job`
maybe better?